libnavui-car/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
Should also add
`src/main/assets/sdk_versions/*`

Reply:
Thanks!